### ISSUE TITLE
Card design licence?

### ISSUE THREAD

**Issue by gingerbeardman** [SheldonSChen/hanafuda#6]
Are your card designs covered by the MIT licence?

Thanks!

**Comment by drazil100** [comment]
I would also like to know this.

**Comment by SheldonSChen** [comment]
I’m not sure, but given that the MIT license is originally meant for code, I’d err on the safer side and say no. Not to mention that they are all heavily based on preexisting artwork.

**Comment by drazil100** [comment]
The designs are old enough that they would be public domain. Specific implementations of designs would fall under copyright. If you drew these cards and didn't straight up trace them off another set then you own the rights and can license them however you like. 

Assuming you made these yourself are you ok with other people using them?

**Comment by SheldonSChen** [comment]
Yes, so long as it is NOT for financial profit.

**Comment by drazil100** [comment]
Thanks a lot! 

Truthfully my friend has already been using them in a project under the assumption they were covered under this project's MIT license (since you had not specified otherwise). We have no intention of making any money off your cards or our project.

One other thing I should ask though is if you are ok with modifications to your cards? I personally made some changes to the colors to make them more vibrant (since MIT allows for modifications) but if this is not ok with you feel free to let us know and we can revert to the original copies of the cards.

**Sample of modifications**
![Screenshot from 2020-11-21 20-39-06](https://user-images.githubusercontent.com/17794391/99892327-7b666c00-2c39-11eb-91b5-7d3897180188.png)

(If you have any questions feel free to ask)

**Comment by SheldonSChen** [comment]
Yup, modifications are completely fine. (Also, your screenshot looks great!)

**Comment by ahigerd** [comment]
Much appreciated. May I suggest [CC-BY-NC](https://creativecommons.org/licenses/by-nc/2.0/) as a way of implementing the terms you want?

**Comment by selfthinker** [comment]
Technically the cards are currently covered by the MIT licence. If you add a different license to the cards later then that would only cover their license from that moment onwards. But the older (including the current) version of the cards are still MIT licensed. You cannot change a license retrospectively.
